src/providers/user/user-provider.js

```diff
@@ -109,6 +135,27 @@ export default class UserProvider extends EventEmitter {
         return success;
     }
 
+    async getStatusForMissionRole(role) {
+        const missionStatusTelemetryObject = await this.missionStatus.getTelemetryObjectForRole(role);
+        if (this.unsubscribeMissionStatus[role] === undefined) {
+            this.unsubscribeMissionStatus[role] = this.realtimeTelemetryProvider.subscribe(missionStatusTelemetryObject, (datum) => {
+                this.emit('missionRoleStatusChange', {
```

Comment:
Hmm. `missionRoleStatusChange` seems like a lot. Is this event indicating both `role` and `status` changes?

Reply:
Nope! It's a change in the Mission Role's status.